packages/model/src/gp2/output.ts

```diff
@@ -133,16 +135,26 @@ export type AuthorUpsertDataObject =
   | { userId: string; externalUserId?: undefined }
   | { externalUserId: string; userId?: undefined };
 
-export type OutputCreateDataObject = OutputCoreObject & {
+export type OutputCreateDataObject = Omit<
+  OutputCoreObject,
+  'relatedOutputs' | 'relatedEvents'
+> & {
   authors: AuthorUpsertDataObject[];
   createdBy: string;
   workingGroupId?: string;
   projectId?: string;
+  relatedOutputs: string[];
```

Comment:
this is an array of strings, these strings being.. IDs, right? In which case I think you should follow our normaln convention and call them `relatedOutputIds` or `relatedEventIds`